src/NuGet.Clients/NuGet.CommandLine/Commands/VerifyCommand.cs

```diff
@@ -36,7 +36,7 @@ public override Task ExecuteCommandAsync()
 
             if (string.IsNullOrEmpty(PackagePath))
             {
-                throw new ArgumentNullException(nameof(PackagePath));
+                throw new ArgumentException(nameof(PackagePath));
```

Comment:
This is technically not correct use of the exception... the parameter here is actually "message", so there's no argument given. But this isn't an argument anyway....?! I think it's probably overkill to refactor this.

Reply:
Are comfortable changing the exception type?

Reply:
I guess `CommandException` or `ArgumentException` really don't matter much as long as we have some user friendly message. Can we add a resource entry for this case and use that for `message` parameter for `ArgumentException` constructor? This should address your [first ](https://github.com/NuGet/NuGet.Client/pull/3799/files#r544566747) comment on this thread.
https://github.com/NuGet/NuGet.Client/blob/a52c0addf437f4e358f786acadd4c91a2c80f9b6/src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx#L5493-L5497

Reply:
You have some evidence that CommandException is more of a standard. How about we merge this PR, and have a separate issue to switch that one?

Reply:
https://github.com/NuGet/Home/issues/10391